analyses/create-corplots/01-process-data.Rmd

```diff
@@ -0,0 +1,270 @@
+  filter(Deceased == "Deceased") %>%
+  write_tsv(file.path(results_dir, "genomic_assays_matched_time_points_list_merge.tsv"))
+```
+
+# Autopsy samples 
+We will focus on patient cases containing autopsy samples and save these maf files separately.
+We will do this for different pairs.
+
+
+```{r maf_files_Deceased}
+#let us filter by Deceased samples
+maf_files_Deceased <- genomic_assays_matched_time_points_list_merge %>% 
+  left_join(snv, by=c('Kids_First_Biospecimen_ID')) %>%
+  filter(!is.na(Kids_First_Participant_ID)) %>%
+  write_tsv(file.path(scratch, "maf_files_Deceased.tsv"))
+```
+
+## Let's save the maf files per each category
+
+```{r save-maf-files}
+##########################################################################################
+# Storing all Dx-Deceased Sample pairs in a list so each file can be processed iteratively
+scratch_Dx_Dec <-
+  file.path(scratch, "Dx_Dec")
+if (!dir.exists(scratch_Dx_Dec)) {
+  dir.create(scratch_Dx_Dec)
+}
+
+setwd(scratch_Dx_Dec)
+data <- maf_files_Deceased %>% 
+  mutate(data = case_when(grepl("Deceased, Diagnosis", descriptors) ~ "Dx_Dec")) %>%
+  filter(data == "Dx_Dec")
+
+# we will split maf data by the "Kids_First_Participant_ID" column
+data %>% 
+  group_split(Kids_First_Participant_ID) %>% 
+  walk(~write_csv(.x, paste0(.x$Kids_First_Participant_ID[1], "-vaf.csv")))
+
+
+##########################################################################################
+# Storing all Pro_Dec Sample pairs in a list so each file can be processed iteratively
+scratch_Pro_Dec <-
+  file.path(scratch, "Pro_Dec")
+if (!dir.exists(scratch_Pro_Dec)) {
+  dir.create(scratch_Pro_Dec)
+}
+
+setwd(scratch_Pro_Dec)
+data <- maf_files_Deceased %>% 
+  filter(descriptors %in% c("Deceased, Progressive", "Deceased, Progressive, Progressive"))
+
+# we will split maf data by the "Kids_First_Participant_ID" column
+data %>% 
+  group_split(Kids_First_Participant_ID) %>% 
+  walk(~write_csv(.x, paste0(.x$Kids_First_Participant_ID[1], "-vaf.csv")))
+
+
+##########################################################################################
+# Storing all Deceased, Recurrence Sample pairs in a list so each file can be processed iteratively
+scratch_Rec_Dec <-
+  file.path(scratch, "Rec_Dec")
+if (!dir.exists(scratch_Rec_Dec)) {
+  dir.create(scratch_Rec_Dec)
+}
+
+setwd(scratch_Rec_Dec)
+data <- maf_files_Deceased %>% 
+  filter(descriptors == "Deceased, Recurrence")
+
+# we will split maf data by the "Kids_First_Participant_ID" column
+data %>% 
+  group_split(Kids_First_Participant_ID) %>% 
+  walk(~write_csv(.x, paste0(.x$Kids_First_Participant_ID[1], "-vaf.csv")))
+
+##########################################################################################  
+# Storing all Dx_Pro_Dec Sample pairs in a list so each file can be processed iteratively
+scratch_Dx_Pro_Dec <-
+  file.path(scratch, "Dx_Pro_Dec")
+if (!dir.exists(scratch_Dx_Pro_Dec)) {
+  dir.create(scratch_Dx_Pro_Dec)
+}
```

Comment:
move to top - honestly, I don't think you need to create more than one folder here - maybe just one folder for the module itself ?